kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/KieServerRouter.java

```diff
@@ -107,10 +126,42 @@
     public KieServerRouter() {
         configRepositoryServiceLoader.forEach(repo -> repository = repo);
         log.info("KIE Server router repository implementation is " + repository);
+        MANAGEMENT_SECURED = Boolean.parseBoolean(System.getProperty(KIE_ROUTER_MANAGEMENT_SECURED, "false"));
```

Comment:
Is there a specific reason to keep the initialization of attributes in constructor and not among other constants?

Reply:
yes... base test class provided by QE does not allow to process thos evariables during class loading initialization.

Reply:
I see, thanks.